examples/tests/test_PredCollapsing.py

```diff
@@ -0,0 +1,74 @@
+import pytest
+import rdflib
+from ipyradiant import collapse_preds
+from rdflib import Graph, URIRef
+from rdflib.extras.external_graph_libs import rdflib_to_networkx_graph
+
+
+@pytest.fixture(scope="module")
+def test_graph():
+    graph = Graph()
+    graph.add(
+        (
+            rdflib.URIRef("www.testing.com/Luke"),
+            rdflib.URIRef("www.testing.com/hates"),
+            rdflib.URIRef("www.testing.com/TottenhamHotspur"),
+        )
+    )
+    graph.add(
+        (
+            rdflib.URIRef("www.testing.com/Luke"),
+            rdflib.URIRef("www.testing.com/has"),
+            rdflib.Literal("scarves"),
+        )
+    )
+    graph.add(
+        (
+            rdflib.URIRef("www.testing.com/Luke"),
+            rdflib.URIRef("www.testing.com/wants"),
+            rdflib.Literal("two jackets"),
+        )
+    )
+    graph.add(
+        (
+            rdflib.URIRef("www.testing.com/Luke"),
+            rdflib.URIRef("www.testing.com/goes"),
+            rdflib.URIRef("www.testing.com/EmiratesStadium"),
+        )
+    )
+    return graph
+
+
+def test_correct_length(test_graph):
+    preds_to_collapse = [
+        rdflib.URIRef("www.testing.com/has"),
+        rdflib.URIRef("www.testing.com/goes"),
+    ]
+    subjects = [
+        rdflib.URIRef("www.testing.com/EmiratesStadium"),
+    ]
+    netx_version = rdflib_to_networkx_graph(test_graph)
```

Comment:
ended up staying with this one because:
1) didnt see a huge difference or gain of information in this one vs. di or multidi graph,
2) with this one all the edges are still in tact so i dont think there is any information lost, and 
3) this is what we currently use for datashader /cytoscape vis so i thought it might be best to stay uniform.

that being said it might be a productive discussion to have or at least something to keep in mind.